Updating the Readme.md

GBruening opened the pull request:
Probably read for typos?

Updated the readme to include the downloading and setup of the git repository by the group lead.

This should include most of the breakout1.rmd from the breakout1 branch.

alcantarar committed:
background for breakout1

alcantarar committed:
add branch creation, cloning, and commit info to breakout_1

alcantarar committed:
finish up push section for breakout1

alcantarar committed:
include viewing commit on github

GBruening committed:
Adding the pictures for adding collaborators, creating new repo, and downloading it as a zip.

GBruening committed:
Added picture for being able to add a file or group of files

GBruening committed:
Update README.md

GBruening committed:
Update README.md

GBruening committed:
Update README.md

Fixing formatting. Pictures to close to text.

GBruening committed:
Update README.md

Adding for help on creating the repo.

GBruening committed:
Update README.md

alcantarar commented on README.md:
```diff
@@ -54,4 +58,33 @@ To download MATLAB go [here](https://www.mathworks.com/downloads/).
+To download the Anaconda distribution, go [here](https://www.anaconda.com/products/individual) and choose your installer near the bottom. Anaconda comes with a few possible IDE's, we prefer VScode but you can use whichever to edit the scripts and run them.
+
+## 4. Creating the Git Repository for your group.
+This section should cover how the group leader will download the repository, create their own version of it, and then invite their group members (collaborators) to the project. Only one person per group needs to do this.
```
add something like "if you haven't been asked to do this, don't." so it's very clear that those who "need to do this" know that they need to do it.

alcantarar replied:
or use some ❗ or 🚨 emojis. Need it to be very clear that 99% of the attendees should NOT do this. could consider hiding it in the wiki.

GBruening replied:
Emoji's.... i'ma go with no lol.

alcantarar commented on README.md:
```diff
@@ -54,4 +58,33 @@ To download MATLAB go [here](https://www.mathworks.com/downloads/).
+
+![](media/new_repo.png)
+
+Title this however you prefer, and make sure the repository is public. 
```
Make name consistent so that we can just directly reference it in our instructions. `ASB_Tutorial_Example` or something.

alcantarar reviewed (changes requested):
🚨 🚨 You can't just take over the `breakout_1` branch! 🚨 🚨 

Changes to the README are good, but your `setup` branch has my `breakout_1` branch as a parent, so my `breakout_1` commits are being included in this PR. Let's **not** do that because my breakout_1 changes aren't ready for a PR. Make a new PR that only has changes related to the` README.md` so I can keep developing `breakout_1` and not split its progress between two PRs. To do this, you should probably just make a new branch off master and then add in your media files and the updated `README.md` from this PR.

Also the title of this PR refers to a "setup.md" that i don't think actually exists.

GBruening committed:
Removing breeakout1 instructions

GBruening committed:
Removing breakout 1 stuff

GBruening commented:
I think I fixed it. But gotta check it real quick

It's gonna still have your commits, but the only thing you changed in those commits was the one md file that I removed.

GBruening committed:
Removing typos in Readme

GBruening commented:
Nevermind i see what you wanted.
